solvis_graphql_api/composite_solution/composite_rupture_detail.py

```diff
@@ -97,72 +122,80 @@ class Meta:
         rupt = rupture_detail(root.model_id, root.fault_system, root.rupture_index)
-        return round(float(rupt['Length (m)'].iloc[0] / 1e3), 0)
+        return round(float(rupt["Length (m)"].iloc[0] / 1e3), 0)
 
     def resolve_rake_mean(root, info, *args, **kwargs):
         rupt = rupture_detail(root.model_id, root.fault_system, root.rupture_index)
-        return round(float(rupt['Average Rake (degrees)'].iloc[0]), 1)
+        return round(float(rupt["Average Rake (degrees)"].iloc[0]), 1)
 
     def resolve_rate_weighted_mean(root, info, *args, **kwargs):
         # with pd.option_context('display.float_format', '${:.2e}'.format):
         rupt = rupture_detail(root.model_id, root.fault_system, root.rupture_index)
         # rupt['rate_weighted_mean'] = rupt.map('${:,.2f}'.format)
         # print('RRR', rupt['rate_weighted_mean'].map('{:,.2e}'.format))
-        return float(rupt['rate_weighted_mean'].iloc[0])
+        return float(rupt["rate_weighted_mean"].iloc[0])
 
     def resolve_rate_max(root, info, *args, **kwargs):
         rupt = rupture_detail(root.model_id, root.fault_system, root.rupture_index)
-        return float(rupt['rate_max'].iloc[0])
+        return float(rupt["rate_max"].iloc[0])
 
     def resolve_rate_min(root, info, *args, **kwargs):
         rupt = rupture_detail(root.model_id, root.fault_system, root.rupture_index)
-        return float(rupt['rate_min'].iloc[0])
+        return float(rupt["rate_min"].iloc[0])
 
     def resolve_rate_count(root, info, *args, **kwargs):
-        with pd.option_context('display.float_format', '${:,.5f}'.format):
+        with pd.option_context("display.float_format", "${:,.5f}".format):
             rupt = rupture_detail(root.model_id, root.fault_system, root.rupture_index)
-            return rupt['rate_count'].iloc[0]
+            return rupt["rate_count"].iloc[0]
 
     def resolve_fault_surfaces(root, info, style, *args, **kwargs):
-        log.info(f'resolve resolve_fault_surfaces : {root.model_id}, {root.fault_system} style: {style}')
+        log.info(
+            f"resolve resolve_fault_surfaces : {root.model_id}, {root.fault_system} style: {style}"
+        )
         composite_solution = get_composite_solution(root.model_id)
-        rupture_surface_gdf = composite_solution._solutions[root.fault_system].rupture_surface(root.rupture_index)
+        rupture_surface_gdf = composite_solution._solutions[
+            root.fault_system
+        ].rupture_surface(root.rupture_index)
 
         rupture_surface_gdf = rupture_surface_gdf.drop(
             columns=[
-                'key_0',
-                'fault_system',
-                'Rupture Index',
-                'rate_max',
-                'rate_min',
-                'rate_count',
-                'rate_weighted_mean',
-                'Magnitude',
-                'Average Rake (degrees)',
-                'Area (m^2)',
-                'Length (m)',
+                "key_0",
```

Comment:
what linting tool do you use to make all quotation marks consistent?

Reply:
@chrisdicaprio that's the black formatter at work. As it's matured the styles have seen a few changes.